### ISSUE TITLE
Dropdown is hidden on last rows - height of HOT is not increased when opening dropdown

### ISSUE THREAD

**Issue by dkwebtec** [handsontable/handsontable#7069]
When setting container height, dropdown is hidden on last rows - height of HOT is not increased when opening dropdown.

### Demo
https://jsfiddle.net/DK568/03df4a2x/5/

### Your environment
* Handsontable version: 7.4.2
* Browser Name and version: Chrome 83
* Operating System: Win10

**Comment by dkwebtec** [comment]
Interesting is that filter header dropdown gets out of the container but cell dropdown not

**Comment by AMBudnik** [comment]
That's true. We have a similar issue here https://github.com/handsontable/handsontable/issues/3828 but it relates to the `preventOverflow`.

**Comment by dkwebtec** [comment]
any workaround? how can I set similar z-index to the cell dropdown like header dropdown has?

**Comment by AMBudnik** [comment]
There's no official API that gives you control over the direction of opening a dropdown list or any other menu.

**Comment by emanuallan** [comment]
This seems to be a consistent issue over the past 5 years, has a good workaround been made? Don't mean to come off rude, just haven't found a good solution.

**Comment by pnowak** [comment]
Hi @emanuallan 

Currently, we try to release stable 8.0.0 version of Handsontable. 

After this, our devs team with business decide which issues we will take care of.  
But I'm not sure that this type of feature we wanted to have.

**Comment by dkwebtec** [comment]
tested in 8.0 and still the same.
if column type dropdown is on the last row on the end of the visible area the dropdown is opened up.
column type handsontable is opening down and not visible

**Comment by AMBudnik** [comment]
@dkwebtec this issue was outside the scope of [v8.0.0 issue list](https://handsontable.com/docs/8.0.0/tutorial-release-notes.html).

**Comment by dkwebtec** [comment]
yes but there is a progress. in 7.4.2 autocomplete/dropdown were also opening down and not visible. Now they open up. If the same CSS will be applied to HOT dropdown it will solve the problem.

**Comment by AMBudnik** [comment]
@dkwebtec Could you share a scenario that you found improved for v8? I've replaced `handsontable` type with `dropdown` for the last column in your demo https://jsfiddle.net/pn5xh96g/1/ + updated the example to v8 but it seems to work the same way.

**Comment by dkwebtec** [comment]
I see now the dropdown in last coumn shows up only on last two rows. And it's same in 7.4.2
OK hope there will be some progress in the future.

**Comment by AMBudnik** [comment]
I will update you as soon as I get any news on that subject @dkwebtec

**Comment by Geeklas** [comment]
I have the same problem. Dropdowns are cut off where the table stops

**Comment by mealplana** [comment]
Same issue here, isn't there a simple css workaround to make the dropdown visible?

**Comment by AMBudnik** [comment]
@mealplana theoretically you can add. But it may not look/work well in some of the cases.

```js
.handsontable .handsontableEditor {
  position: fixed;
}
```

https://jsfiddle.net/handsoncode/896h7onm/

**Comment by mealplana** [comment]
Thanks, I think this may work in our case

**Comment by AMBudnik** [comment]
Related https://github.com/handsontable/handsontable/issues/5506

**Comment by AMBudnik** [comment]
This issue is still replicable using 12.1.0 version https://jsfiddle.net/5wv9hyqk/.

**Comment by AMBudnik** [comment]
Hi @mealplana @dkwebtec 

Thank you for sharing the issue report.

I have great news. This issue is solved in v16.0.0, which was released just now!

If you have any questions, please feel free to write to us at support@handsontable.com. Here you can read more about the changes https://handsontable.com/docs/javascript-data-grid/changelog/#_16-0-0
